velox/exec/Driver.cpp

```diff
@@ -500,7 +512,18 @@ StopReason Driver::runInternal(
           return stop;
         }
 
-        auto op = operators_[i].get();
+        auto* op = operators_[i].get();
+
+        if (FOLLY_UNLIKELY(shouldYield())) {
```

Comment:
There is an existing mechanism to yield that relies on Task::shouldStop(). Are we going to keep both mechanisms or remove the existing one?

Reply:
@mbasmanova chat offline, we will remove the exiting one if the new one works for both batch and interactive workloads. Thanks for pointing out!